prodtests/full_system_test.sh

```diff
@@ -81,21 +86,19 @@ if [ "0$GENERATE_ITS_DICTIONARY" == "01" ]; then
 fi
 
 mkdir -p raw
-HBFUTILPARAMS="HBFUtils.nHBFPerTF=${NHBPERTF};HBFUtils.orbitFirst=0;"
```

Comment:
I'd like to keep the `HBFUtils.maxNOrbits` setting actually in case `ALLOW_MULTIPLE_TF != 1`. Can I get that by different means? Otherwise I'd prefer to remove only nHBFPerTF and orbitFirst. Or will this also be propagated automatically from the digitizer?

Reply:
yes, the whole HBFUtils is extracted from the `o2secondary-vertexing-workflow_configuration.ini` (or whatever you provide via `--hbfutils-config` available for all converters). So, just set the maxNOrbits during digitization.

Reply:
OK, thx, makes sense.

One more question though: what happens if we run multiple digitizers, using the `--incontext collisioncontext.root` option.
Will it also read the HBFUtils settings? Or will it overwrite the ini file with the settings from the command line?
This will not be a problem here, since you set it to both digitizer invocations, I am just asking in general.

Reply:
Currently the o2-sim-digitization writes its INI file before the SimReader initialization (whose option is the `--incontext`), so, the `--configKeyValues HBFUtils...` should be provided for every digitization stage as the INI file from eventual previous stage is ignored and overwritten. 
Perhaps we should 
1) if the `--incontext` is provided, read the previous INI file in the beginning of `defineDataProcessing`
2)  move storing the INI file to the very end of `defineDataProcessing` and avoid it if `--incontext` is provided.
@sawenzel, what do you think?

Reply:
> Perhaps we should
> 
>     1. if the `--incontext` is provided, read the previous INI file in the beginning of `defineDataProcessing`
> 
>     2. move storing the INI file to the very end of `defineDataProcessing` and avoid it if `--incontext` is provided.
>        @sawenzel, what do you think?

I like this approach. I mean, one could still override settings for the second digitizer on the command line. But by default, one would have the same settings, and one would also not overwrite them so easily.

Reply:
That seems to be reasonable. (But I guess we are using the previously written file only when someone is saying `--configFile`. Is this the case?)

Reply:
Yes (but it is not yet there). I hope we will not hit again the problem of loading the whole content of the INI file with plenty of "unknown" params inside, since only those which are linked to digitization workflow will be found there...

Reply:
Now I am confused. If we merge this PR, will the raw converters read the ini files by default or now?

Reply:
yes, I was answering to @sawenzel  concerning the reading of INI file by digitizer in `--configFile` case.